### PR TITLE
Refactoring/aws/kms

### PR DESCRIPTION
**Pull request by misg**
This PR aims to migrate private AWS KMS service to the new architecture, as part of #183. Please look at the [proprietary PR](https://github.com/nccgroup/ScoutSuite-Proprietary/pull/117) as well.

### CODE REVIEW

File: ScoutSuite/providers/aws/facade/facade.py [hunk 1]
```diff
@@ -18,7 +18,8 @@
 
 try:
     from ScoutSuite.providers.aws.facade.config_private import ConfigFacade
-except:
+    from ScoutSuite.providers.aws.facade.kms_private import KMSFacade
+except NameError:
```
Comment (Aboisier) [inline ScoutSuite/providers/aws/facade/facade.py]: Shouldn't this be `ImportError`?

Comment (Aboisier) [reply ScoutSuite/providers/aws/facade/facade.py]:
```suggestion
except ImportError:
```

Comment (Aboisier) [reply ScoutSuite/providers/aws/facade/facade.py]: Can confirm.

Comment (misg) [reply ScoutSuite/providers/aws/facade/facade.py]: Good catch, thanks!

**Pull request merged**